docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -214,7 +214,36 @@ For more information about devfile contents requirements, see the "What is outer
 +
 **Solution:** Select a predefined runtime type from the **Review and configure for deployment** view. Make sure you meet all of the {ProductName} <<repository requirements>> for the runtime you select, or configure your own <<custom build and deployment instructions>>.
 
-[role="_additional-resources"]
-.Additional resources
-For information about importing and configuring code to {ProductName} using the CLI, see link:https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/getting-started/getting_started_in_cli/#getting-started-in-the-cli[Getting started in the CLI
-].
\ No newline at end of file
+=== Building with Red Hat Container Registry tokens
+
+To improve your application-building experience, configure a Red Hat Container Registry token.  
+
+.Procedure
+
+Create the Red Hat Container Registry token by following these steps:
+
+. Go to link:https://access.redhat.com/terms-based-registry/#/[registry service account]. 
+. Create a new registry service account by clicking *New Service Account*. 
+. Complete the `Name` and `Description` field. 
+. For `Token Information`, click `Docker Configuration`. 
```

Comment:
should there be a step before this one (new step 4) that says "Click **Create**."?